components/dashboard/tailwind.config.js

```diff
@@ -5,96 +5,90 @@
  */
 
 // tailwind.config.js
-const colors = require('tailwindcss/colors');
+const colors = require("tailwindcss/colors");
 
 module.exports = {
     jit: true,
-    purge: [
-        './public/**/*.html',
-        './src/**/*.{js,ts,tsx}',
-    ],
+    purge: ["./public/**/*.html", "./src/**/*.{js,ts,tsx}"],
     important: true,
-    darkMode: 'class',
+    darkMode: "class",
     theme: {
         extend: {
             colors: {
                 gray: colors.warmGray,
                 green: colors.lime,
                 orange: colors.amber,
                 blue: {
-                    light: '#75A9EC',
-                    DEFAULT: '#5C8DD6',
-                    dark: '#265583',
+                    light: "#75A9EC",
+                    DEFAULT: "#5C8DD6",
+                    dark: "#265583",
                 },
-                'gitpod-black': '#161616',
-                'gitpod-gray': '#8E8787',
-                'gitpod-red': '#CE4A3E',
-                'gitpod-kumquat-light': '#FFE4BC',
-                'gitpod-kumquat': '#FFB45B',
-                'gitpod-kumquat-dark': '#FF8A00',
-                'gitpod-kumquat-darker': '#f28300',
-                'gitpod-kumquat-gradient': 'linear-gradient(137.41deg, #FFAD33 14.37%, #FF8A00 91.32%)',
+                "gitpod-black": "#161616",
+                "gitpod-gray": "#8E8787",
+                "gitpod-red": "#CE4A3E",
+                "gitpod-kumquat-light": "#FFE4BC",
+                "gitpod-kumquat": "#FFB45B",
+                "gitpod-kumquat-dark": "#FF8A00",
+                "gitpod-kumquat-darker": "#f28300",
+                "gitpod-kumquat-gradient": "linear-gradient(137.41deg, #FFAD33 14.37%, #FF8A00 91.32%)",
             },
             container: {
                 center: true,
             },
             outline: {
-                blue: '1px solid #000033',
+                blue: "1px solid #000033",
+            },
+            width: {
+                120: "28rem",
+                128: "32rem",
```

Comment:
This file only adds these two new definitions, since I tried with [`w-[28rem]`](https://tailwindcss.com/docs/width#arbitrary-values) but not worked.

I can revert changes and add these four lines then save without format if you want.

Reply:
Hey, many thanks for contributing larger widths 💯 I think this will be very useful.

Also, no worries about the formatting -- looks good to me, thanks for doing this as well. 👍

Small nit: Shouldn't `120: "28rem"` actually be `112: "28rem"`? (Seems like in Tailwind v2, width/height units are usually `rem * 4`)